pages/Advanced Queries.md

```diff
@@ -39,22 +39,87 @@ description:: Advanced queries are written with [Datalog](https://en.wikipedia.o
 	  #+END_TIP
 	- ### Query Inputs
 	  These are special values for a query's `:inputs` key above:
-		- Page and block inputs
+		- #### Special page and block inputs
 			- `:current-page` - Name of current page. See [this example](((63b70dc8-58a5-4a43-ae19-28143edb7752)))
 			- `:current-block` - `:db/id` of current block. See [this example](((63bc5e11-24f1-45fd-945d-4a272e5ecf0d)))
 			- `:parent-block` - `:db/id` of parent block
-		- Inputs for querying journal page blocks and rules like `between`
+		- #### Special date inputs (useful for querying journal pages or blocks relative to today's date)
```

Comment:
Headers have permanent links based on the first line of content so it'd be good to move descriptions down a line to have a readable and more stable link. See https://docs.logseq.com/#/page/publishing/block/available%20urls for more. I can fix this